### PR TITLE
My first pull request

### PR DESCRIPTION
**Pull request by StarryIce**
I have changed the readMe file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# pull-requests-demo
\ No newline at end of file
+# pull-requests-demo
+Hello, I am learning.
```
Comment (anishmunirathinam) [inline README.md:2]: change hello to hi!

**Review by anishmunirathinam — CHANGES_REQUESTED** [review]
Needs some change, please check.

File: README.md (hunk 1 above)
Comment (StarryIce) [reply README.md:2]: @anishmunirathinam Hello is fine and more opt solution.

**Review by anishmunirathinam — APPROVED** [review]
LGTM!

**Pull request merged**